tests/test_aaa_authorization_service.rb

```diff
@@ -22,54 +22,72 @@ class TestAaaAuthorizationService < CiscoTestCase
   def setup
     super
     feature_tacacs
+    preconfig_tacacs_server_access(tacacs_groups[0])
```

Comment:
The creation of the tacacs-server is needed for the remaining tests, but my point is that it should be fairly trivial to combine these two into a single setup method.